### ISSUE TITLE
TypeError: null is not an object (evaluating 'RNBackgroundGeolocation.ready')

### ISSUE THREAD

**Issue by reaganscofield** [transistorsoft/react-native-background-geolocation#1622]
## Your Environment
* Plugin version: 4.9.4
* Platform:  Android
* OS version:
* Device manufacturer / model: android emulator 
* React Native version : 0.68.2
* Plugin config
*
```javascript <!-- Syntax-highlighting:  paste your code below -->
import BackgroundGeolocation, { Config } from "react-native-background-geolocation";

const config: Config = {
    // Geolocation Config
    desiredAccuracy: BackgroundGeolocation.DESIRED_ACCURACY_HIGH,
    distanceFilter: 10,
    // Activity Recognition
    stopTimeout: 5,
    // Application config
    debug: true, // <-- enable this hear sounds for background-geolocation life-cycle.
    logLevel: BackgroundGeolocation.LOG_LEVEL_VERBOSE,
    stopOnTerminate: false,   // <-- Allow the background-service to continue tracking when user closes the app.
    startOnBoot: true,        // <-- Auto start tracking when device is powered-up.
}

export async function initBackgroundFetch() {
    console.log("does this function get invoked?")
    BackgroundGeolocation?.ready(config).then((state) => {
        console.log("START", state)
    });
}
```

## Expected Behavior
expecting log the state but that code does not run at all 

## Actual Behavior

<img width="723" alt="Screenshot 2023-01-12 at 14 22 45" src="https://user-images.githubusercontent.com/29593266/212065457-5a5b9dc5-c94a-44b5-aff2-7b7f55c275f9.png">

## Steps to Reproduce

1.
2.
3.
4.

## Debug logs

<details><summary>Logs</summary>

``` <!-- Syntax highlighting:  DO NOT REMOVE -->
PASTE_YOUR_LOGS_HERE
```

</details>

**Comment by christocracy** [comment]
Your app doesn’t think the plugin exists.

I suggest you restart your dev server.

**Comment by christocracy** [comment]
You did not follow the **Required Setup Instructions** in the readme.

**Comment by christocracy** [comment]
>I did skip the liking setup because it does suggest doing that for react-native >= 0.60 and my installed react is 0.68.2

Is your version `68.1` *greater than or equal to* `60.2`?  Yes or no?

Every step is **required**.

**Comment by christocracy** [comment]
>in the terms of license setup, it suggests that I can do debug without a license I only need license for production release

The plugin is fully function in debug builds so you can try before you buy, in spite of "*License Validation Failure*" messages from the plugin.

**Comment by reaganscofield** [comment]
okay thanks, let me add the linking I will update you on how it goes here 
thank you again for your help much appreciated it.

**Comment by reaganscofield** [comment]
okay thanks, let me add the linking I will update you on how it goes here 
thank you again for your help much appreciated it.

**Comment by reaganscofield** [comment]
@christocracy thank you, everything works as expected, is there a way to track user's location when the app is completely killed?

**Comment by christocracy** [comment]
> is there a way to track user's location when the app is completely killed?

See api docs Config.stopOnTerminate.

also see *HTTP Guide*

**Comment by christocracy** [comment]
This is now fixed in `4.9.5` with a new [`type MotionActivity`](https://transistorsoft.github.io/react-native-background-geolocation/interfaces/motionactivity.html) introduced for [`Location.activity`](https://transistorsoft.github.io/react-native-background-geolocation/interfaces/location.html#activity).

The API docs are now correct.
